samples/nrf9160/mqtt_simple/Kconfig

```diff
@@ -33,8 +33,19 @@ config MQTT_PAYLOAD_BUFFER_SIZE
 	int "MQTT payload buffer size"
 	default 128
 
+config MQTT_TLS_SEC_TAG
+	int "TLS credentials security tag"
+	default 24
+
+config MQTT_TLS_SESSION_CACHING
+	bool "Enable TLS session caching"
+
+config MQTT_TLS_PEER_VERIFY
+	int "Set peer verification level"
+	default 2
+
 endmenu
 
 menu "Zephyr Kernel"
-source "$ZEPHYR_BASE/Kconfig.zephyr"
+source "Kconfig.zephyr"
```

Comment:
Why this change? I don't think it should be in here.

Reply:
The NordicBuilder suggested this change.

Reply:
Then I guess it's fine :+1: just looked like a weird change. I guess it's because the Kconfig of the sample is quite old.